Task10_Auth/src/test/java/DeleteUsersTest.java

```diff
@@ -0,0 +1,75 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import reseponsesDTO.UsersDto;
+
+import java.util.List;
+import java.util.Random;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class DeleteUsersTest {
+    Random random = new Random();
+
+    @BeforeEach
+    public void precondition() {
+        UsersClient.createUserIfNotExist();
+    }
+
+    @Test
+    public void deleteUserTest() {
+        List<UsersDto> usersList = (List<UsersDto>) UsersClient.getUsers().getResponseBody();
+        UsersDto usersDto = usersList.get(random.nextInt(usersList.size()));
+
+        ResponseWrapper deleteUserResponse = UsersClient.deleteUser(usersDto);
+
+        List<UsersDto> userListAfterDelete = (List<UsersDto>) UsersClient.getUsers().getResponseBody();
+        List<String> zipCodesAfterDelete = (List<String>) ZipCodesClient.getZipCodes().getResponseBody();
+        assertAll("204, user removed, zip code is back",
+                () -> assertEquals(204, deleteUserResponse.getResponseCode()),
+                () -> assertFalse(userListAfterDelete.contains(usersDto), "removed user doesn't exist in users list"),
+                () -> assertTrue(zipCodesAfterDelete.contains(usersDto.getZipCode()), "removed user zip code added to the zip codes list")
```

Comment:
Just pay attention on your messages on failure, they are strange. This message appears if assertion fails and with current messages it is mess. I expect "Removed user **exists** in users list" and so on

Reply:
resolved